src/components/SimpleTable.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from 'clsx';
+
+export interface Props {
+  title?: string;
+  footer?: JSX.Element;
+  option?: JSX.Element;
+  content: JSX.Element;
+  className?: string;
+  height?: string;
+}
+
+export default function SimpleTable(props: Props): JSX.Element {
```

Comment:
Always name props interface with the name of the component + suffix Props. 
```suggestion
export default function SimpleTable(props: SimpleTableProps): JSX.Element {
```

Please change this everywhere.